### PR TITLE
Update python docker base image from slim-buster to slim-bullseye

### LINKED ISSUE #926: Upgrade fidesops base image from Debian Buster to Bullseye
fidesops is currently based on python:3.x.y-slim-buster. In August 2022 Debian Buster (10) is [scheduled](https://wiki.debian.org/DebianReleases#Production_Releases) to move from official (oldstable) support to [LTS](https://wiki.debian.org/LTS) community support.

The fidesops base image should be changed to python:3.x.y-slim-bullseye to ensure that timely security and version updates are applied to the application image.

### PR DESCRIPTION
**Pull request by sanders41**
# Purpose

Update the base python image in docker from slim-buster to slim-bullseye.

I ran `make build`, `make pytest`, `make pytest-integration`, `make reset-db`, `make user`, `make quickstart`, and `make server`. All ran without error.

# Changes
- Update the base python image in docker from slim-buster to slim-bullseye
- Remove ipython

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #926

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,7 +25,10 @@ The types of changes are:
 * Access and erasure support for Adobe Campaign [#905](https://github.com/ethyca/fidesops/pull/905)
 
 ### Changed
+
 * Users should be able to click on the full field of a dropdown-type filter to open up the dropdown [#747](https://github.com/ethyca/fidesops/pull/903)
+* Updated the python docker base image from slim-buster to slim-bullseye [928](https://github.com/ethyca/fidesops/pull/928)
+* Removed ipython from the docker install [928](https://github.com/ethyca/fidesops/pull/928)
```
Comment (seanpreston) [inline CHANGELOG.md:31]: This seems to have happened in a previous change, or not been committed here? I'm still seeing [ipython in dev-requirements.txt](https://github.com/ethyca/fidesops/blob/main/dev-requirements.txt#L3).

Comment (sanders41) [reply CHANGELOG.md:31]: I didn't look at dev-requirements. That means we had ipython in two places, it was also installed with `apt-get`. Do we want to keep it as a dev requirement? I asked in stand up and everyone said they don't use it so unless you think we should keep it I can remove it there also.

Comment (sanders41) [reply CHANGELOG.md:31]: I went ahead and removed it, but if you want it back just let me know and I'll do that.

Comment (seanpreston) [reply CHANGELOG.md:31]: I use it a lot in local dev, but it's not something we need to include in the Docker containers, happy for it to go so folks can make the choice if they want it.

**Review by seanpreston — APPROVED** [review]

**Pull request merged**